### PR TITLE
develop navbar

### PR DESCRIPTION
**Pull request by Wence17**

### CODE REVIEW

File: app/components/Navbar.tsx [hunk 1]
```diff
@@ -1,11 +1,41 @@
...
   return (
     <div className="bg-white h-[50px] flex items-center justify-center ">
-      <h1>Veritas Payment Portal</h1>
-      <h3></h3>
-
+        
+      <nav className="w-full bg-white shadow-md">
+        <div className="justify-between px-4 mx-auto lg:max-w-7xl md:items-center md:flex md:px-8">
+          <div className="flex items-center justify-between py-3 md:py-5 md:block">
+            <a href="#">
```
Comment (KMF21) [inline app/components/Navbar.tsx:12]: In next js we make use of Link tags, not anchor tags.

File: app/components/Navbar.tsx [hunk 2]
```diff
@@ -1,11 +1,41 @@
...
+          <div>
+            <div
+              className={`flex-1 justify-self-center pb-3 mt-8 md:block md:pb-0 md:mt-0 `}
+            >
+              <ul className="items-center justify-center space-y-8 md:flex md:space-x-6 md:space-y-0">
+                
+                <li>
+                  <Link href="#" className='flex space-x-4 items-center'>
+                    
+                    <Image className="w-10 h-10 rounded-full dark:bg-gray-600" src="" loading="lazy" alt="Rounded avatar" width="1280" height="100"/>
```
Comment (KMF21) [inline app/components/Navbar.tsx:27]: Wence I made mention of using a Tailwind avatar for the profile picture.

File: app/components/Navbar.tsx (hunk 1 above)
Comment (KMF21) [reply app/components/Navbar.tsx:12]: please let's keep to using Link tags for all links

**Review by KMF21 — APPROVED** [review]

**Comment by KMF21** [comment]
well-done @Wence17! It would do us good if we pay close attention to our issues when taking them on.

**Comment by KMF21** [comment]
Again please always link your pull request to the Issue it solves. eg "close @214" research to know how to link an issue within a pull request. Thank you

**Comment by Wence17** [comment]
Ok noted!

**Pull request merged**